functions.php

```diff
@@ -0,0 +1,21 @@
+<?php
+
+require_once ('index.php');
+
+/*
+ * displays each film and its attributes (from the database) in a human readable way
+ *
+ * @param array which is an array containing arrays for each film
+ *
+ * return string displaying films and film details
+ */
+function displayFilms(array $array) : string {
+    $string = '';
+    foreach ($array as $item) {
```

Comment:
`$films as $film`

Reply:
updated to $films as $film, also updated the string to get rid of extraneous ' ' around html tags.